src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar-form search-bar-big"
```

Comment:
If you use modifiers, use like this.
Also you do not need to add redundant prefix search-bar
```suggestion
      class="form form--big"
```